mmdet3d/core/bbox/structures/utils.py

```diff
@@ -167,6 +168,8 @@ def mono_cam_box2vis(cam_box):
     Returns:
         :obj:`CameraInstance3DBoxes`: Box after conversion.
     """
+    warning.warn('DeprecationWarning: The hack for visualizing monocular 3D '
```

Comment:
The warning is incorrect, we should directly say that this function will be deprecated soon.